WLC/web_endpoint.py

```diff
@@ -28,6 +33,7 @@ def api_upload_image():
     else:
         return render_template('upload_test.html')
 
+
 @app.route("/api/resubmit_code", methods=['POST', 'GET'])
```

Comment:
I don't know the exact thinking behind this but we can add an `UPDATE` or `PUT`to the `/api/upload_image` endpoint and combine the two functions.

Reply:
I think it should be separate because the functionality is different.